plugins/moonphase/moonphase.plugin.zsh

```diff
@@ -0,0 +1,54 @@
+# -----------------------------------------------------------------------------
+#          FILE: moonphase.plugin.zsh
+#   DESCRIPTION: The current moon phase for your terminal prompt. Uses a very 
+#                simple calculation based upon a known new moon date. Sorry
+#                time travelers, this won't work before 1970. 
+#                Just drop '$(printmoon)' into your PROMPT where you want the
+#                moon emoji to appear.
+#        AUTHOR: Sean Carolan (scarolan[at]gmail.com)
+#       VERSION: 1.0.0
+# -----------------------------------------------------------------------------
+
+function printmoon() {
+  # Accepts a time in days between 0 and 29.53
+  # Defaults to the current moon phase
+  if (($+1)); then
+    phase=$1
```

Comment:
This and the other variables in `printmoon()` and `calcphase()` look like they should be `local`.